styles/custom.css

```diff
@@ -54,10 +54,18 @@ body {
 }
 
 .heroContainer::before {
+  top: 25%;
+  left: 0;
+  background: var(--brand-faded-7);
+}
+@media (media-width: 768px){
```

Comment:
Found a typo here.
It should be `@media (min-width)`, not `@media (media-width)`.

Replace the word *media-width* with *min-width*